cmd/apiserver/app/server/options.go

```diff
@@ -41,6 +41,7 @@ type ServiceCatalogServerOptions struct {
 	EtcdOptions *EtcdOptions
 	// TPROptions are options for serving with TPR as the backing store
 	TPROptions *TPROptions
+	StopCh     <-chan struct{}
```

Comment:
I'd think that the best place would be in `NewCommandServer` if doing clean up were possible. I've been trying to execute code after RunServer terminated via modifying Run, adding PostRun, and adding PersistentPostRun. None of the code seems to execute, in which case there's no point having the stop channel created so soon. What is the expected method for terminating the apiserver? Perhaps I'm doing it wrong.